### PR TITLE
fix/packetSpam&more

### PR DESCRIPTION
**Pull request by avcio9**

### CODE REVIEW

**Comment by avcio9** [comment]
It seems like sending Npcs queued allows us to do it while client is loading without breaking the collisions

File: src/packets/ClientProtocol/ClientProtocol_1080/clientUpdate.ts [hunk 1]
```diff
@@ -234,7 +234,19 @@ export const clientUpdatePackets: any = [
       ],
     },
   ],
-  ["ClientUpdate.UpdateManagedLocation", 0x112300, {}],
+  [
+    "ClientUpdate.UpdateManagedLocation",
```
Comment (QuentinGruber) [inline src/packets/ClientProtocol/ClientProtocol_1080/clientUpdate.ts:238]: What's the purpose of this packet?

Comment (avcio9) [reply src/packets/ClientProtocol/ClientProtocol_1080/clientUpdate.ts:238]: Teleprt observer vehicle

**Review by QuentinGruber — APPROVED** [review]
Didn't tested it in game

**Comment by avcio9** [comment]
> Didn't tested it in game

its on kraków since yesterday

**Review by Z1Meme — APPROVED** [review]

**Pull request merged**